tests/test_resources.py

```diff
@@ -244,6 +245,16 @@
         resource_type='dataflow.googleapis.com/Job',
         name='//dataflow.googleapis.com/projects/my_project/locations/us-central1/jobs/my_resource'
     ),
+    ResourceTestCase(
+        resource_data={
+            'name': test_resource_name,
+            'location': 'us-central1',
+            'project_id': test_project
+        },
+        cls=GcpMemorystoreRedis,
+        resource_type='redis.googleapis.com/Instance',
+        name='//redis.googleapis.com/projects/my_project/locations/us-central1/instances/my_resource'
```

Comment:
Have you tested exporting the resource? The documentation claims the format is 

`//redis.googleapis.com/projects/PROJECT_NUMBER/locations/LOCATION/nodes/NODE_ID` but lists the type as `redis.googleapis.com/Instance`. Thats a deviation from their typical format, and I think the one you used is more likely. We should confirm what CAI actually does, and if necessary ask them to update their documentation.

https://cloud.google.com/asset-inventory/docs/resource-name-format

Reply:
Did a cai export in my account and it seems to line up 

```
{
  "name": "//redis.googleapis.com/projects/cleardata-lab-bretttest/locations/us-central1/instances/testing",
  "asset_type": "redis.googleapis.com/Instance",
  "resource": {
```